charts/component-charts/ondat-operator/templates/operator.yaml

```diff
@@ -57,7 +58,7 @@ spec:
         resources:
           limits:
             cpu: 250m
-            memory: 200Mi
+            memory: 512Mi
```

Comment:
Why does it so much more ?

Reply:
We've seen some edge cases in which newly added features of the operator can require larger than usual burst memory

We're looking into how to curtail this increase in a future release but for now we've increased the limit to prevent OOM-killing

The average memory footprint of the operator has not changed, hence the unchanged `requests`